horace_core/sqw/@rundatah/rundatah.m

```diff
@@ -157,10 +164,16 @@ function clear_det_cache()
             else
                 proj_mode = 2;
             end
-            % Calculate projections            
+            % Calculate projections
             [u_to_rlu,pix_range,pix] = obj.calc_projections_(obj.detdcn_cache,[],proj_mode);
         end
+        function flds = indepFields(obj)
+            flds = indepFields@rundata(obj);
+            flds = [flds(:)',{'transform_sqw'}];
```

Comment:
Does the orientation of the fields actually matter? why not just
```matlab
flds = [flds, 'transform_sqw'];
```

Reply:
Interesting. Apparently it looks like it works here. Probably it did not work in a previous Matlab versions